airflow/utils/log/logging_mixin.py

```diff
@@ -90,6 +90,14 @@ def __init__(self, logger, level):
         self.level = level
         self._buffer = ''
 
+    def close(self):
+        """
+        Provide close method, for compatibility with the io.IOBase interface.
+
+        This is a no-op method.
+        """
+        pass
```

Comment:
```suggestion
```

Not needed (and sadly not liked by pylint) because the doc-comment is a statement.

Or open a PR to disable the `unnecessary-pass` rule globally, as I do like the "clarity" of an explicit pass here.

Reply:
I agree, I prefer an explicit `pass`, but I also don't want to figure out pylint configs today, so I'll take the suggestion this time round.